src/Grid/Grid.js

```diff
@@ -24,13 +25,20 @@ class Grid extends React.Component{
     page: this.props.page,
     rowsPerPage: this.props.rowsPerPage,
     data: this.props.data,
-    columns: this.props.columns
+    serverUrl: this.props.serverUrl,
+    records: []
```

Comment:
So, now we have data and records? Why not only one?

Reply:
I removed it, but for future PR (not of my own): By this time there is an other branch that uses the variable `data` only for testing purposes. The other branch doesn't have the capability to populate the `<Grid />` component via URL